### PR TITLE
Michael/aiports popularity

### PR DESCRIPTION
**Pull request by s3419069**
[AB#104](https://dev.azure.com/Tringo/cabcbfb6-3a22-44cf-ac98-b3810925a870/_workitems/edit/104)

[AB#111](https://dev.azure.com/Tringo/cabcbfb6-3a22-44cf-ac98-b3810925a870/_workitems/edit/111)

### CODE REVIEW

File: Tringo.AirportsParser/Program.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+namespace Tringo.AirportsParser
+{
+    class Program
+    {
+        private const string WIkiAirportsAddress = "https://en.wikipedia.org/wiki/List_of_international_airports_by_country";
+        private const string outputFileName = "AirportsPassengers.json";
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
```
Comment (rmit-s3668804-Dzmitry-Kakaruk) [inline Tringo.AirportsParser/Program.cs]: What is this "Hello World" in the production code?

Comment (rmit-s3668804-Dzmitry-Kakaruk) [reply Tringo.AirportsParser/Program.cs]: Also, if you want to debug, don't use console, use logger

File: Tringo.AirportsParser/Program.cs [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                            var airportData = new AirportsData
+                            {
+                                IATACode = iataText,
+                                NumberofPassengers = passengers
+                            };
+                            resultList.Add(airportData);
+                        }
+                        else
+                        {
+                            //throw new Exception("Parsing error");
```
Comment (rmit-s3668804-Dzmitry-Kakaruk) [inline Tringo.AirportsParser/Program.cs]: Maybe put a logger here?

File: Tringo.AirportsParser/Program.cs (hunk 1 above)
Comment (s3419069) [reply Tringo.AirportsParser/Program.cs]: It's one time running code, who cares. I'd easier drop it rather than refactor ;)

Comment (s3419069) [reply Tringo.AirportsParser/Program.cs]: I already added generated file, so it's just for reporting

**Pull request merged**